composer.json

```diff
@@ -16,13 +16,13 @@
 
     "require": {
         "php":                           ">=5.3.1",
-        "behat/mink":                    "~1.6@dev",
+        "behat/mink":                    "~1.7@dev",
```

Comment:
Not sure, that this is needed, because driver will use tests from separate package and won't be too sad about fact, that same tests are also present as part of Mink test suite.